src/platform_impl/android/ndk_glue.rs

```
@@ -436,6 +473,198 @@ pub unsafe fn onWindowFocusChanged(
   wake(event);
 }
 
+#[allow(non_snake_case)]
+pub unsafe fn onNewIntent(env: JNIEnv, _: JClass, intent: JObject) {
```

There are a number of silent failure paths in this function e.g.

```rust
let Ok(action) = env.get_string(&action)
  .map(|action| action.to_string_lossy().to_string())
else {
  return; // Silent failure
};
```

Should these errors be propagated or logged?

---

[`2268fac`](https://github.com/tauri-apps/tao/pull/1155/commits/2268fac61116813c2f8c4b04f30a050b37803c84)